exporter/signalfxexporter/translation/constants.go

```diff
@@ -516,20 +516,31 @@ translation_rules:
 
 # Translations to derive Network I/O metrics.
 
-## Calculate network.total.
+## Calculate extra network I/O metrics system.network.packets.total and system.network.io.total.
 - action: copy_metrics
   mapping:
-    system.network.io: network.total
-  dimension_key: direction
-  dimension_values:
-    receive: true
-    transmit: true
+    system.network.packets: system.network.packets.total
+    system.network.io: system.network.io.total
+- action: aggregate_metric
+  metric_name: system.network.packets.total
+  aggregation_method: sum
+  without_dimensions:
+  - device
+- action: aggregate_metric
+  metric_name: system.network.io.total
+  aggregation_method: sum
+  without_dimensions:
+  - device
+
+## Calculate extra network.total metric.
+- action: copy_metrics
+  mapping:
+    system.network.io.total: network.total
```

Comment:
Actually one question, does this need to necessarily need to depend on system.network.io.total or could it be kept independent and still use `system.network.io`? When rules start depending on previous rules things get a bit more complicated I think.

Reply:
That's a good point. I'll change it accordingly